OMS.Core/Entities/Employee.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace OMS.Core.Entities
+{
+    public class Employee
+    {
+        [Key]
+        public int empId { get; set; }
+
+        [Required,MaxLength(25)]
+        public string empFname { get; set; }
+        public string empLname { get; set; }
+        public string empAddress { get; set; }
+        public DateTime empBirthdate { get; set; }
+        public Gender empGender { get; set; }
```

Comment:
do not add emp suffix. we already know that this is employee entity. why do you need to add a suffix of emp? and please use FirstName not Fname or Lname